src/StreetNameRegistry.Api.BackOffice/Infrastructure/MigrationsHelper.cs

```diff
@@ -33,31 +33,10 @@ public static class MigrationsHelper
                 .Execute(() =>
                 {
                     logger?.LogInformation("Running EF Migrations.");
-                    RunInternalSequence(sequenceConnectionString, loggerFactory);
                     RunInternalBackOffice(backOfficeConnectionString, loggerFactory);
                 });
         }
 
-        private static void RunInternalSequence(string connectionString, ILoggerFactory? loggerFactory)
```

Comment:
Does this exist in the lambda project?

Reply:
Yes , see https://github.com/Informatievlaanderen/streetname-registry/blob/main/src/StreetNameRegistry.Api.BackOffice.Handlers.Lambda/Function.cs#L88

Reply:
No, not yet. We'll want to run them at the end of [Function.ConfigureServices()](https://github.com/Informatievlaanderen/streetname-registry/blob/main/src/StreetNameRegistry.Api.BackOffice.Handlers.Lambda/Function.cs#L108)?

Reply:
I would do this in the `ctor` and leave `ConfigureServices` purely for registration